packages/ng/dialog/directives/dialog-open.directive.ts

```diff
@@ -1,29 +1,26 @@
-import { Directive, HostListener, inject, Input, TemplateRef } from '@angular/core';
+import { Directive, inject, input, TemplateRef } from '@angular/core';
 import { provideLuDialog } from '../dialog.providers';
 import { LuDialogService } from '../dialog.service';
 import { LuDialogConfig } from '../model';
 
 @Directive({
 	selector: '[luDialogOpen]',
 	providers: [provideLuDialog()],
+	host: {
+		'(click)': 'click()',
+	},
 })
 export class DialogOpenDirective {
 	#dialogService = inject(LuDialogService);
 
-	@Input({
-		required: true,
-		alias: 'luDialogOpen',
-	})
-	dialog: TemplateRef<void>;
+	readonly dialog = input.required<TemplateRef<void>>({ alias: 'luDialogOpen' });
 
-	@Input()
-	luDialogConfig: LuDialogConfig<unknown>;
+	readonly luDialogConfig = input();
```

Comment:
```suggestion
	readonly luDialogConfig = input<LuDialogConfig<unknown>>();
```

this way, we can remove the `as` in the `click()` method